changelogs/current.yaml

```diff
@@ -82,6 +82,11 @@ minor_behavior_changes:
 
 bug_fixes:
 # *Changes expected to improve the state of the world and are unlikely to have negative effects*
+- area: config
+  change: |
+    Preserve presence information for the :ref:`connect_config
+    <envoy_v3_api_field_config.route.v3.RouteAction.UpgradeConfig.connect_config>` field, so that
+    CONNECT request handling is consistent with the documentation.
```

Comment:
mind actually inlining the change to avoid folks having to do the lookup?

Reply:
Sure, added some more explanation. I also moved this up into the `minor_behavior_changes` section as it is definitely a change in behavior. (But happy to move it back if you prefer.)